src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -20,32 +21,18 @@
     [utils.re-frame :as rf]
     [utils.worklets.communities :as worklets]))
 
-;; NOTE: values compared against `scroll-amount` to trigger animations.
-(def expand-header-threshold
-  "Dragging distance to collapse/extend the community."
-  150)
-
-(def sheet-displacement-threshold
-  "Dragging distance to round sheet borders and move the sheet 8 units."
-  (+ expand-header-threshold 20))
+(def snap-header-threshold-factor
+  "Threshold to automatically move the header to a collapsed/expanded state and avoid an
+  intermediate state. Applied to `collapse-threshold`."
+  0.65)
 
-(def text-movement-threshold
-  "Dragging distance to start the text movement from/to the bottom to/from the right."
-  (* expand-header-threshold 0.7))
+(def navbar-content-threshold-factor
+  "When the community name and logo start to appear. Applied to sheet-displacement-threshold."
+  32)
 
-(def info-opacity-threshold
+(def info-opacity-threshold-factor
   "Dragging distance to appear/disappear the community info (description, tags & stats)."
-  (* expand-header-threshold 0.5))
-
-(def snap-header-threshold
-  "Threshold to automatically move the header to a collapsed/expanded state and avoid an
-  intermediate state."
-  (* expand-header-threshold 0.75))
-
-(def expand-header-limit
-  "Max dragging distance where the header animation ends. It works to identify when to
-  start the flat-list scrolling."
-  (+ sheet-displacement-threshold 56))
+  0.5)
```

Comment:
These values were fixed, they implied a constant scroll gesture to collapse the community. but sometimes it felt strange, because a height of 500 was covered with a gesture of 150.

This PR had to move these values to be dynamic, that's why this is changing too much and also the worklets. The benefit is that now the gesture is 1:1 and feels very good